rad/src/operators/mod.rs

```diff
@@ -152,6 +152,24 @@ pub fn identity(input: RadonTypes) -> Result<RadonTypes, RadError> {
     Ok(input)
 }
 
+/// Check if the operator can be used in a tally script.
+///
+/// Some operators cannot be used in the tally stage because they are considered to be
+/// "consensus unsafe".
+///
+/// For example, sorting the input is forbidden.
```

Comment:
I would like to explain why, @aesedepece is something like this ok?

Because the tally result should not depend on the order, so sorting it is a logic error. Sorting tally inputs could produce mistakes in reputation redistribution.